pywbem/cim_obj.py

```diff
@@ -1946,6 +1963,7 @@ def tocimxmlstr(self, indent=None):
         :class:`~pywbem.CIMClass` object, as a :term:`unicode string`.
 
         The returned CIM-XML representation is consistent with :term:`DSP0201`.
+        It does not represent the class path of this class.
```

Comment:
Same as above about representation, etc.

Reply:
Done - changed to "include"